### PR TITLE
Sort of 3 optionlist at failsafe tab

### PR DESCRIPTION
**Pull request by HThuren**
Use of common sortSelect

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/failsafe.js
Comment (chmelevskij) [inline src/js/tabs/failsafe.js:272]: Could we follow similar patter to above of using `name` instead of `id`? so we don't need extra property to keep in sync and this would look like:
```suggestion
        $('select[name"gps_rescue_sanity_checks"]').sortSelect();
```

File: src/js/tabs/failsafe.js
Comment (chmelevskij) [inline src/js/tabs/failsafe.js:253]:
```suggestion
        $('select[name="failsafe_switch_mode"]').sortSelect();
```

File: src/tabs/failsafe.html
Comment (chmelevskij) [inline src/tabs/failsafe.html:50]: We don't need special ids for this. In general `name` is a good unique property and is required for accesibility etc.
```suggestion
                                <select class="switchMode" name="failsafe_switch_mode">
```

**Review by chmelevskij — APPROVED** [review]
I don't think ids are necessary and for consistency using name would be better.

**Pull request merged**
